x/incentive/types/keys.go

```diff
@@ -17,11 +17,20 @@ const (
 	QuerierRoute = ModuleName
 )
 
+// TODO: Could store previous reward time accruals under the same key indexed by type ('usd_minting', 'hard_supply', etc.)
+
 // Key Prefixes
 var (
-	ClaimKeyPrefix  = []byte{0x01} // prefix for keys that store claims
-	BlockTimeKey    = []byte{0x02} // prefix for key that stores the blocktime
-	RewardFactorKey = []byte{0x03} // prefix for key that stores reward factors
+	USDXMintingClaimKeyPrefix                       = []byte{0x01} // prefix for keys that store USDX minting claims
```

Comment:
Store structure looks fine. Generically, for each incentive, there's:
* A key that stores the claims
* A key that stores the global accumulator value
* A key that stores the last time the accumulator was updated.

One option would be to combine the last two keys into a struct and just update the struct whenever the accumulator is updated, but I'll leave that as an implementation detail and don't think it's important to change.

Reply:
Added a TODO to refactor towards this structure